### ISSUE TITLE
How do you calculate loss on training of the Hybrid_ViT heatmap

### ISSUE THREAD

**Issue by mucunwuxian** [anguyen8/face-vit#3]
Thank you for your great work!

I would like to ask.
Especially in the case of authentication based on similarity as in test.py.
<img width="200" alt="image" src="https://github.com/anguyen8/face-vit/assets/9916906/cc64b1d0-15ba-4184-be68-aeda7b3c9dbe">

How do you calculate loss on training?
The distance calculation in test.py was as follows.
Maybe it's not a cosine distance, but is the ArcFace loss function used?
```
diff = np.subtract(embeddings1, embeddings2)
dist = np.sum(np.square(diff), 1)
```

Even if it's a rough answer, I appreciate it.
Best regards,

**Comment by anguyen8** [comment]
@mucunwuxian Thank you for your question!

Yes, the `ArcFace` loss is used during training. Please see the paper (Eq. 7, Sec. 3.2).

**Comment by mucunwuxian** [comment]
@anguyen8 
Thank you for your reply!
I'll try using the information you provided.👍
